nuxeo-core/nuxeo-core-storage-mongodb/src/main/java/org/nuxeo/ecm/core/storage/mongodb/GridFSBinaryManager.java

```diff
@@ -138,17 +150,16 @@ public Binary getBinary(Blob blob) throws IOException {
             return super.getBinary(blob); // just open the stream and call getBinary(InputStream)
         }
         // we already have a file so can compute the length and digest efficiently
-        File file = ((FileBlob) blob).getFile();
+        File file = blob.getFile();
         String digest;
         try (InputStream in = new FileInputStream(file)) {
             digest = DigestUtils.md5Hex(in);
         }
         // if the digest is not already known then save to GridFS
-        GridFSDBFile dbFile = gridFS.findOne(digest);
+        GridFSFile dbFile = gridFSBucket.find(Filters.eq("filename", digest)).first();
```

Comment:
`"filename"` seems to be used many times, please use a constant for it. Isn't it defined by GridFS itself?